src/amo/components/Errors/ErrorComponent/index.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+import * as React from 'react';
+import makeClassName from 'classnames';
+import NestedStatus from 'react-nested-status';
+
+import Card from 'ui/components/Card';
+
+import './styles.scss';
+
+type Props = {|
+  children: React.Node,
+  className?: string,
+  code: number,
+  header: React.Element<any> | string,
+|};
+
+type InternalProps = {|
```

Comment:
We don't really need an `InternalProps` that is just the same as `Props`. Just use `Props` in the `class` declaration below.